spec/requests/api/v1/provider_spec.rb

```diff
@@ -129,7 +128,6 @@ def get_provider_codes_from_body(body)
                region_code: :south_west,
                accrediting_provider: "N",
                scheme_member: "N",
-               last_published_at: DateTime.now.utc,
```

Comment:
We don't know what the effect this will have on UCAS. I'm not sure we should remove it.

Reply:
This was only for the test setup.